src/guide/chapters/core-language.md

```diff
@@ -217,10 +218,10 @@ It is often useful to update the values in a record.
 
 ```elm
 > { bill | name = "Nye" }
-{ age = 57, name = "Nye" }
+{ name = "Nye", age = 57 } : { age : number, name : String }
 
 > { bill | age = 22 }
-{ age = 22, name = "Gates" }
+{ name = "Gates", age = 22 } : { name String, age : number }
```

Comment:
Again, verbatim copy of the REPL output, which somehow inverted the type annotation's fields.